source/margarita/states.rst

```diff
@@ -0,0 +1,621 @@
+Salt States
+===========
+
+SLS files
+---------
+
+.. _base:
+
+base
+~~~~
+
+Installs packages like `build-essential` and `sudo` that all
+our servers need.
+
+No configuration.
```

Comment:
I like being explicit about this. Can we include this on other states that have no configuration?